lib/cosmos/config/config_parser.rb

```diff
@@ -143,6 +144,18 @@ def error(message, usage = "", url = @url)
       return Error.new(self, message, usage, url)
     end
 
+    # Called by the ERB template to render a partial
+    def render(template_name, options = {})
+      b = binding
+      if options[:locals]
+        options[:locals].each do |key, value|
+          eval("@#{key} = #{value}", b)
```

Comment:
In rails, locals don't get an @ symbol applied to them.  Don't think that is necessary